deployment/arm/deploy-cosmosDB.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd "$parent_path"
+
+businessUnit=$1
+appName=$2
+env=$3
+primaryRegion=$4
+secondaryRegion=$5
+tertiaryRegion=$6
+
```

Comment:
should verify that above parameters are set and exit if not defined

Reply:
As per the discussion with Wallace, deleting this script and will deploy the Cosmos as part of the global deployment.